funding/manager_test.go

```diff
@@ -1167,7 +1167,11 @@ func assertChannelAnnouncements(t *testing.T, alice, bob *testNode,
 				baseFee := aliceCfg.DefaultRoutingPolicy.BaseFee
 				feeRate := aliceCfg.DefaultRoutingPolicy.FeeRate
 
-				require.EqualValues(t, 1, m.MessageFlags)
+				msgflags := lnwire.ChanUpdateRequiredMaxHtlc
+				if unannounced {
+					msgflags |= lnwire.ChanUpdateDontForward
+				}
```

Comment:
do you need an `else` condition to ensure we zero the `ChanUpdateDontForward` bit if the channel *is* announced?